roles/rsyslog/roles/input_roles/basics/defaults/main.yaml

```diff
@@ -12,6 +12,7 @@ rsyslog_unprivileged: false
 #       Note: unles rsyslog_send_over_tls_only is set to true, insecure network connection
 #       is also allowed.
 rsyslog_capabilities: []
+rsyslog_basics_packages: ['rsyslog-gnutls']
```

Comment:
is the user supposed to override this? If not, it should go to vars, not defaults, and be prefixed with double underscore.

Reply:
Rsyslog supports openssl, as well.  But it looks the plugin is not included in the RHEL/CentOS distribution.  So, there's no chance to override for now.  I will follow your advice.  Thanks!

Reply:
Note: removed the var.